qcodes/__init__.py

```diff
@@ -4,13 +4,13 @@
 
 # config
 
-from qcodes.config import Config
+import qcodes.configuration as qcconfig
 from qcodes.logger import start_all_logging
 from qcodes.logger.logger import conditionally_start_all_logging
 from qcodes.utils.helpers import add_to_spyder_UMR_excludelist
 from .version import __version__
 
-config: Config = Config()
+config: qcconfig.Config = qcconfig.Config()
```

Comment:
is the type annotation still needed here? :(
```suggestion
config = qcconfig.Config()
```

Reply:
yes

Reply:
ok, I lack some mypy understanding

Reply:
I honestly don't know why but mypy fails without it